### PR TITLE
Adding instructions on setting up the project

### PR DESCRIPTION
**Pull request by lucywyman**

### CODE REVIEW

File: source/development_process/index.rst [hunk 1]
```diff
@@ -85,12 +85,25 @@ Determine Requirements
 
 Setting Up the Project
 ----------------------
-- Create the git repo
-- Create the issue tracker
+
+**Create the git repo**: Unless you have special permission in the `OSL
+organization github <https://github.com/osuosl>`_, you will probably
+need to ask a full-time employee to make the repository for you.  We
```
Comment (iankronquist) [inline source/development_process/index.rst]: Please don't use 'we'.

File: source/development_process/index.rst [hunk 2]
```diff
@@ -85,12 +85,25 @@ Determine Requirements
...
+issue tracker for managing issues on a repository (which doesn't
+require any set up). If we are using
+chili project or another issue tracker, you may have to set that
+up separately from the repo. 
 - Create the project skeleton with Django
 - Create the Dockerfile
 - Create a virtualenv
-- Create the projects default branch (if not master)
+**Create the projects default branch (if not master)**: Under the 
+repositories settings (the little screwdrive in the sidebar) you can
```
Comment (iankronquist) [inline source/development_process/index.rst]: *screwdriver

File: source/development_process/index.rst [hunk 3]
```diff
@@ -85,12 +85,25 @@ Determine Requirements
...
+**Create the Issue Traker**: Often times we will simply use the github
+issue tracker for managing issues on a repository (which doesn't
+require any set up). If we are using
+chili project or another issue tracker, you may have to set that
+up separately from the repo. 
 - Create the project skeleton with Django
 - Create the Dockerfile
 - Create a virtualenv
-- Create the projects default branch (if not master)
+**Create the projects default branch (if not master)**: Under the
```
Comment (iankronquist) [inline source/development_process/index.rst]: Can you make these instructions a little more specific? Can you link to github docs describing how to do this.

File: source/development_process/index.rst [hunk 4]
```diff
@@ -85,12 +85,25 @@ Determine Requirements
...
-- Create the git repo
-- Create the issue tracker
+
+**Create the git repo**: Unless you have special permission in the `OSL
+organization github <https://github.com/osuosl>`_, you will probably
+need to ask a full-time employee to make the repository for you.  We
+want all the repositories for the OSL to be owned by the OSL, so don't
+make a repo under your own account! Once the repo is created you'll
+want to use ``$ git clone <SSH URI>`` to start working on the project.
+**Create the Issue Traker**: Often times we will simply use the github
```
Comment (iankronquist) [inline source/development_process/index.rst]: All future projects will use the Redmine issue tracker. Describe how to create a new project on redmine.

File: source/development_process/index.rst [hunk 5]
```diff
@@ -85,12 +85,25 @@ Determine Requirements
...
+issue tracker for managing issues on a repository (which doesn't
+require any set up). If we are using
+chili project or another issue tracker, you may have to set that
+up separately from the repo. 
 - Create the project skeleton with Django
```
Comment (iankronquist) [inline source/development_process/index.rst]: Please cover these bullet points as well.

**Comment by iankronquist** [comment]
Can you use the following header style? It will be more constent with the rest of the document.
```
Create the git repo
~~~~~~~~~~~~~~~
lorem ipsum
```

File: source/development_process/index.rst (hunk 1 above)
Comment (lucywyman) [inline source/development_process/index.rst]: I read the little instructions. I would argue this makes the document feel very impersonal and scientific in an undesirable way -- we aren't business majors here -- but if you insist.

**Comment by lucywyman** [comment]
Yes, would you like me to change all the other instances of this throughout the document which you evidently didn't catch as well?

**Comment by lucywyman** [comment]
I would also like to point out that the "Writing Chef Tests" section often uses "you", and also uses the bolded sections.

File: source/development_process/index.rst (hunk 5 above)
Comment (lucywyman) [inline source/development_process/index.rst]: I will, they just require research and I am leaving soon.

**Comment by iankronquist** [comment]
I'm fixing other people's errors. It's a slog.

**Comment by lucywyman** [comment]
@iankronquist Perhaps you could just ask them to fix their own errors?

**Pull request merged**